storagemarket/impl/client.go

```diff
@@ -338,6 +338,8 @@ func (c *Client) ProposeStorageDeal(ctx context.Context, params storagemarket.Pr
 		return nil, xerrors.Errorf("computing commP failed: %w", err)
 	}
 
+	pricePerEpoch := big.Mul(big.NewInt(int64(pieceSize.Padded())), params.Price)
```

Comment:
Pretty sure `params.Price` is per GB here

Reply:
based on how this gets into the actors proposal object, i don't think it is.